### ISSUE TITLE
Ensure keyboard navigation is possible across editor and dashboard

### ISSUE THREAD

**Issue by bmattb** [GoogleForCreators/web-stories-wp#2757]
## Feature Description

As part of our continuing efforts to ensure accessibility standards are met we need to check that the user can use only a keyboard across all editor and dashboard sections. This will highlight any areas of concern.

## Additional Context

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by o-fernandez** [comment]
@pbakaus is working on an accessibility brief, which should cover the minimum reqs for this.

**Comment by bmattb** [comment]
@pbakaus brief is required as per @o-fernandez  comment above, please review.

**Comment by pbakaus** [comment]
Well, guidelines [exist](https://docs.google.com/document/d/15FHCukTDrWf7xYpPkBG3bGRgRQZeqzkCcglaNXgaw9s/edit), but this issue is so broad that it is not super useful maybe.  Would be great for @carlos-kelly and @barklund to break this up into individual tickets around dashboard and the editor.

**Comment by pbakaus** [comment]
also, @swissspidy as cc, who I made the owner around a11y for now

**Comment by o-fernandez** [comment]
Turned this into an Epic and moved to the Epics column. It's lacking details on what's needed, is it covered by other tickets? @barklund @carlos-kelly @swissspidy ?

**Comment by swissspidy** [comment]
There's actually already an epic for _Accessibility Review for Stable_, #117. It already has some tickets attached to it, so I suggest closing this one as a duplicate and concentrate our efforts there.

I will open more tickets under that epic to address keyboard navigation. In a first step, we would need to test & document the status quo, so I am going to focus on that.

Duplicate of #117
